### PR TITLE
API: list all mandatory fields for FindingCreateSerializer

### PR DESCRIPTION
**Pull request by XiChen-Tibco**
Fix the required parameter issue within swagger for issue #3121.

### CODE REVIEW

File: dojo/api_v2/serializers.py [hunk 1]
```diff
@@ -1058,8 +1058,13 @@ class FindingCreateSerializer(TaggitSerializer, serializers.ModelSerializer):
 
     class Meta:
         model = Finding
-        exclude = ['images']
+        fields = '__all__'
         extra_kwargs = {
+            'mitigation': {'required': True},
+            'impact': {'required': True},
```
Comment (Maffooch) [inline dojo/api_v2/serializers.py:1064]: I don't believe these fields are required anywhere in the app anymore, but they were when the issue was created.

**Review by Maffooch — APPROVED** [review]

**Review by valentijnscholten — APPROVED** [review]

**Comment by valentijnscholten** [comment]
Tbh I haven't checked, but thought the '__all__' thing did that. But maybe not since only images was excluded?

**Comment by Maffooch** [comment]
No the current `fileds = __all__` and the old `exlcude = ['images']`are the exact same since images have been removed

**Comment by valentijnscholten** [comment]
yeah that's what I meant

**Pull request merged**